tracing-subscriber/src/filter/directive.rs

```diff
@@ -142,6 +157,15 @@ impl DirectiveSet<StaticDirective> {
             None => false,
         }
     }
+
+    /// Same as `enabled` above, but short circuits to false if the
+    /// `Directive` has fields
```

Comment:
is this correct? i believe `enabled_for` will continue iterating if the `Directive` has fields, rather than short-circuiting --- it will _skip_ directives with fields.